lib/plugins/audit.js

```diff
@@ -110,6 +115,14 @@ function auditLogger(options) {
 
         log.info(obj, 'handled: %d', res.statusCode);
 
+        if (logMetrics && logMetrics instanceof Object) {
```

Comment:
You don't need these checks here if you just assert like the other objects up on line 27